vent/io/config/devices.ini

```diff
@@ -42,6 +42,16 @@ offset_voltage    = 2.2
 output_span       = 2.0
 conversion_factor = 70.3
 
+[oxygen_sensor]
+# Note: calibration must convert to units percent O2
```

Comment:
Will have to table this until we get an oxygen tank in the lab.

Reply:
I guess this isn't critical for now